eq-author/src/App/page/Design/PageHeader.js

```diff
@@ -35,7 +34,6 @@ export class PageHeader extends React.Component {
     showDeleteConfirmDialog: false,
     showMovePageDialog: false,
   };
-  π;
```

Comment:
This made me smile - a single PI on its own. But it wasn't a linting error :D Weird.

Reply:
Was saving that for later